lib/Ohmbrewer_Thermostat.cpp

```diff
@@ -1,52 +1,36 @@
 #include "Ohmbrewer_Thermostat.h"
 #include "Ohmbrewer_Screen.h"
 #include "Ohmbrewer_Publisher.h"
+#include "pid.h"
 
+//Define Variables we'll be connecting to with PID
+double Setpoint, Input, Output;
 
+//Define the aggressive and conservative Tuning Parameters for PID
+double aggKp=4, aggKi=0.2, aggKd=1;
+double consKp=1, consKi=0.05, consKd=0.25;
 
-/**
- * The desired target temperature. Defaults to Celsius
- * @returns The target temperature in Celsius, as a Temperature object pointer
- */
-Ohmbrewer::Temperature* Ohmbrewer::Thermostat::getTargetTemp() const {
-    return _targetTemp;
-}
-
-/**
- * Sets the target temperature
- * @param targetTemp The new target temperature in Celsius
- * @returns The time taken to run the method
- */
-const int Ohmbrewer::Thermostat::setTargetTemp(const double targetTemp) {
-    unsigned long start = millis();
-
-    _targetTemp->set(targetTemp);
-
-    return start - millis();
-}
-
-/**
- * The Thermostat's heating element
- * @returns The heating element
- */
-Ohmbrewer::HeatingElement* Ohmbrewer::Thermostat::getElement() const {
-    return _heatingElm;
-}
+//PID windowing variables
+int WindowSize = 5000;
+unsigned long windowStartTime;
```

Comment:
Also, why are all these variables in lines 6-15 not included under the class? Doing this puts them into the global scope...